hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/impl/FilePerBlockStrategy.java

```diff
@@ -95,15 +94,15 @@ public void writeChunk(Container container, BlockID blockID, ChunkInfo info,
       throws StorageContainerException {
 
     checkLayoutVersion(container);
-
     Preconditions.checkNotNull(dispatcherContext);
     DispatcherContext.WriteChunkStage stage = dispatcherContext.getStage();
 
     if (info.getLen() <= 0) {
       if (LOG.isDebugEnabled()) {
         LOG.debug("Skip writing empty chunk {} in stage {}", info, stage);
       }
-      return;
+      ExitUtils.terminate(1, "Empty Write chunk received " + info,
```

Comment:
why we terminate here? is there something wrong if we just return?

Reply:
Removed the code..